compiler-tools/cfgviz.py

```
@@ -0,0 +1,279 @@
+        result = subprocess.run(cmd, capture_output=True, text=True)
+
+        if result.returncode != 0:
+            print(f"Error running dot: {result.stderr}", file=sys.stderr)
+            sys.exit(1)
+
+        print(f"Success! Generated: {output_file}", file=sys.stderr)
+    finally:
+        # Clean up temp file
+        try:
+            os.unlink(temp_dot_file)
+        except:
```

Ruff (0.14.14)

[error] Do not use bare `except`

(E722)